projects/m1/002-bottle-deposits/solutions/GianlucaGuerra/solution_bottle_deposit.py

```diff
@@ -0,0 +1,10 @@
+#drink container with less or equal to 1 lit. will receive back 0.10$
+#drink container with more than 1 lit. will receive back 0.25$
+#How many bottle of liter do you have?
+print('How many bottles do you want recycle? Please read the following instractions.')
+n=input('Please enter the number of bottles less or equal to 1 lt.: ')
+N=input('Please enter the number of bottles bigger than 1 lt.: ')
```

Comment:
don't use this kind of naming convention.
Not all the languages are case sensitive so they wouldn't understand the difference between n and N.
Other than that, it wouldn't allow you to read your code in the future (especially in bigger projects)